static/app/views/settings/projectSourceMaps/projectSourceMapsArtifacts.tsx

```diff
@@ -180,13 +181,17 @@ export function ProjectSourceMapsArtifacts({params, location, router, project}:
         subtitle={
           <VersionAndDetails>
             {params.bundleId}
-            {tabDebugIdBundlesActive && (
-              <DebugIdBundlesTags
-                dist={debugIdBundlesArtifactsData?.dist}
-                release={debugIdBundlesArtifactsData?.release}
-                loading={debugIdBundlesArtifactsLoading}
-              />
-            )}
+            {tabDebugIdBundlesActive &&
+              // TODO(Pri): Move the loading to the component once fully transitioned to associations.
+              !debugIdBundlesArtifactsLoading &&
+              (debugIdBundlesArtifactsData?.associations ? (
+                <Associations associations={debugIdBundlesArtifactsData?.associations} />
```

Comment:
nit - you checked this exists
```suggestion
                <Associations associations={debugIdBundlesArtifactsData.associations} />
```

Reply:
I prefer to leave it as it is for now... because once we fully transaction this can be `null`